scipy/spatial/transform/rotation.py

```diff
@@ -0,0 +1,862 @@
+from __future__ import division, print_function, absolute_import
+
+import numpy as np
+import scipy.linalg
+import re
+import warnings
+
+
+AXIS_TO_IND = {'x': 0, 'y': 1, 'z': 2}
```

Comment:
Use underscore for consistency?

Reply:
We removed underscore in favor of upper case names in a previous suggestion. Should I change it back?

Reply:
I suggest underscore and upper case as a private constant.